### ISSUE TITLE
[a11y][Toggle Button] - Pressed state only visible via color

### ISSUE THREAD

**Issue by OliKei** [SAP/ui5-webcomponents#8045]
### Bug Description

The toggle button has a pressed state. It is visualized by a different color (blue). The pressed state color (#0163D8) contrasts to the non-pressed state color grey (#BBC3C9) with 3.1:1. However, accessibility requirements state there should be a second channel provided for end users to more easily recognize a state. 
Many standard toggle buttons provide additional indicators, such as checkmarks, LED lights (where the lighted version differs in shape), or filled/unfilled icons. 

The library may offer toggle button examples which actually provide a second channel, so an accessible implementation in end user apps is more probable.

### Affected Component

Toggle Button

### Expected Behaviour

The toggle button allows end users to easily perceive the pressed state, independent from color changes.

### Isolated Example

https://sap.github.io/ui5-webcomponents/playground/?path=/docs/main-togglebutton--docs

### Steps to Reproduce

1. Open the link https://sap.github.io/ui5-webcomponents/playground/?path=/docs/main-togglebutton--docs
2. Move focus onto toggle button
3. Press ENTER or SPACE to toggle
4. view how the state changes (if you cannot see it, watch out for the color of the button's frame).
5. Judge if the perception of the pressed state is fast and unambiguous.
...

### Priority

Medium

### UI5 Web Components Version

1.18.0

### Browser

Chrome, Safari

### Operating System

macOS 14.2.1

### Additional Context

Accessibility
requirement "Use of Color"
https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by NHristov-sap** [comment]
Hello @ui5-webcomponents-topic-b,

I am forwarding this issue to you, because oit seems the Default type Toggle Button toggled/non-focused state background color doesn't match the specification. Please take a look.

Best Regards,
Nikolay Hristov
